source/main/commands/export.ts

```diff
@@ -74,7 +74,27 @@ export default class Export extends ZettlrCommand {
       if (fileDescriptor !== null) {
         exporterOptions.sourceFiles.push(fileDescriptor)
         exporterOptions.cwd = fileDescriptor.dir
-        exporterOptions.targetDirectory = (exportTo === 'temp') ? app.getPath('temp') : fileDescriptor.dir
+        switch (exportTo) {
```

Comment:
Instead of instantiating a dialog here, please use `this._app.getWindowManager().askDir()` with the appropriate options!

Reply:
The issue with askDir() is that it shows a wrong title and button text. Should i expand the function so that i can pass custom texts?

Reply:
> The issue with askDir() is that it shows a wrong title and button text. Should i expand the function so that i can pass custom texts?

Hm, on macOS it doesn't really matter (which is why I didn't pay too much attention to that), but yes. In general I'd like the window functionality to remain inside the BrowserWindow, so if there's no way we could come up with something generic that works, then extending the function is the way to go.